src/panels/config/system-health/ha-config-system-health.ts

```diff
@@ -191,6 +191,17 @@ class HaConfigSystemHealth extends SubscribeMixin(LitElement) {
               </div>
             `
           );
+        } else {
+          keys.push(html`
```

Comment:
This is weird. You should just add it to the system health in the backend instead.

Reply:
@zsarnett Remove this and lets merge this one instead: <https://github.com/home-assistant/core/pull/74318>

Reply:
I tagged https://github.com/home-assistant/core/pull/74318 for the 2022.7 milestone so this one ca safely be merged without users losing access to that data.